docs/snippets/azure/AppHost/storage.module.bicep

```diff
@@ -0,0 +1,67 @@
+@description('The location for the resource(s) to be deployed.')
```

Comment:
How do these files get updated? Manually?

Reply:
As of right now, yes, these are manually updated by running the `dotnet run --publisher manifest`.